bpf/nat/connecttime.go

```diff
@@ -84,50 +85,61 @@ func RemoveConnectTimeLoadBalancer(cgroupv2 string) error {
 	return nil
 }
```

Comment:
Should ```RemoveConnectTimeLoadBalancer``` as well be moved to ```libbpf``` APIs?

Reply:
After some discussion, this is not straight forward and we will leave it out for now. It is only executed when felix restarts and we cannot remember links.